Add --plot-to-board to plot command

Ronaq13 opened the pull request:
This flag of plot command will send the plotted burndown chart to the first
card of the `Done` column.

close #134

matheussbernardo committed:
Add show-backlog command: It creates a command that shows the planning backlog, it closes issue #45

Ronaq13 committed:
Add --plot-to-board to plot command

This flag of plot command will send the plotted burndown chart to the first
card of the `Done` column.

close #134

Ronaq13 committed:
Added truthy check

Added the 'to' statement for the corresponding 'expect' statement
in the unit test of checking the file presence.

close #178

Ana06 commented on lib/cli.rb:
```diff
@@ -189,13 +189,24 @@ def burndown
+    if options[:plot_to_board]
+      begin
+        chart = BurndownChart.new @@settings
+        chart.update(options)
```
This is also modifying the `yml` file, which we don't want to modify here. We only want to print the image and push it to the card.

Ronaq13 replied:
Please help identifying the line at which it modifying the .`yml` file. And by .`yml` file do you mean burndown-data-01.yml file which is created at time of initialization

Ana06 replied:
inside of the `update` method.

Yes, the `burndown-data-i.yaml` file, where i is the number of the sprint

Ronaq13 replied:
@Ana06 I dont see any change in the yml file. Please check now is it good to merge?

cornelius committed:
Fix custom doing columns

Trollolo accepts a definition of custom doing columns in the
burndown YAML file. This used to be part of the `not_done_columns`
setting. This has been split up into `todo_columns` and
`doing_columns`.

Old configurations which use the `not_done_columns` setting will
have to be adjusted. There should be an error message now
explaining what to do.

This fixes a bug where custom doing columns where not properly
accounted for when creating burndown data.

cornelius committed:
Make Rubocop happy

Ana06 commented:
@Ronaq13 @matheussbernardo @cornelius where are those commits coming from?

Ana06 commented:
@Ronaq13 what is the state of this PR? :thinking:

Ana06 commented:
Closing due to inactivity, @Ronaq13 feel free to reopen it if you continue working on it :wink:
